test/e2e/atlas/helper_test.go

```diff
@@ -88,6 +91,140 @@ const (
 	e2eSharedMDBVer    = "5.0"
 )
 
+const (
+	defaultTimeout   = 30 * time.Minute
+	defaultTick      = 1 * time.Minute
+	ResourceNotFound = "RESOURCE_NOT_FOUND"
+)
+
+type K8SHelper struct {
+	atlasClient *mongodbatlas.Client
+	project     *mongodbatlas.Project
+	clusters    map[string]*mongodbatlas.Cluster
+	orgID       string
+	t           *testing.T
+}
+
+func NewK8sHelper(t *testing.T) *K8SHelper {
+	t.Helper()
+
+	// These ENV variables MUST be set before running the test
+	orgID := os.Getenv("MCLI_ORG_ID")
+	privateKey := os.Getenv("MCLI_PRIVATE_API_KEY")
+	publicKey := os.Getenv("MCLI_PUBLIC_API_KEY")
+	opsManagerURL := os.Getenv("MCLI_OPS_MANAGER_URL")
+	transport := digest.NewTransport(publicKey, privateKey)
+	tClient, err := transport.Client()
+	require.NoError(t, err, "can not create transport for atlas client")
+
+	atlasClient := mongodbatlas.NewClient(tClient)
+	atlasClient.BaseURL, err = url.Parse(opsManagerURL)
+	require.NoError(t, err, "can not create atlas client")
+
+	return &K8SHelper{
+		atlasClient: atlasClient,
+		t:           t,
+		orgID:       orgID,
+		clusters:    map[string]*mongodbatlas.Cluster{},
+	}
+}
+
+func (kh *K8SHelper) NewProject(project *mongodbatlas.Project) {
+	kh.t.Helper()
+
+	createdProject, resp, err := kh.atlasClient.Projects.Create(context.Background(), project, &mongodbatlas.CreateProjectOptions{})
+	require.Nil(kh.t, err, "error while creating project", err, resp.Status)
+	kh.project = createdProject
```

Comment:
[q] any reason why you cannot follow a similar approach used by other functions in this file and use the CLI itself instead of defining a client? Please, have a look at

Reply:
discussed offline. This will be addressed in a follow up PR